.github/actions/test-ruby/action.yml

```diff
@@ -40,14 +49,30 @@ runs:
         restore-keys: |
           ${{ runner.os }}-gems-
 
-    - name: .env for ci
+    - name: Run zeitwerk chek
+      if: inputs.ZEITWERK == 'true'
+      run: |
+        bin/rails zeitwerk:check
+      shell: bash
+
+    - name: Run brakeman
+      if: inputs.BRAKEMAN == 'true'
+      uses: devmasx/brakeman-linter-action@v1.0.0
+      env:
+        GITHUB_TOKEN: ${{github.token}}
+
+    - name: Run rubocop
+      if: inputs.RUBOCOP == 'true'
+      uses: gimenete/rubocop-action@1.0
```

Comment:
in prime, there is a git diff in combination to detect which files are changing and only run rubocop for that - https://github.com/signalwire/prime-rails/blob/master/.github/workflows/ci.yml#L30

prime is probably the exception, as we'd generally want to run it for everything.  So, prime could set this to false and run its own maybe.